### ISSUE TITLE
IME popups render below window popups

### ISSUE THREAD

**Issue by YaLTeR** [YaLTeR/niri#125]
![image](https://github.com/YaLTeR/niri/assets/1794388/290ad04a-d874-4a57-b7c9-bd5146970039)

**Comment by aohanhongzhi** [comment]
![Image](https://github.com/user-attachments/assets/b052d831-052a-487b-84d6-51562a5974d1)

**Comment by YaLTeR** [comment]
How is that related to this issue?

**Comment by aohanhongzhi** [comment]
> How is that related to this issue?

**Bug Report: Fcitx5 Pinyin Input Visibility Issue**

**Description**:  
When typing Pinyin, the candidate selection window completely covers the preedit text (raw input letters). Users cannot see what they're currently typing because the visual feedback is obscured.

**Expected Behavior**:  
The preedit text ("nihao") should remain visible either:
- Above the candidate window, or
- Integrated into the candidate panel itself

**Reproduction Steps**:
1. Enable Fcitx5 with Pinyin input method
2. Type any Pinyin (e.g., "nihao")
3. Observe that the candidate box hides the typed letters

**Screenshots**:  

![Image](https://github.com/user-attachments/assets/9240d2a3-5103-4798-8d20-29aa3ab5222e)

**Priority**: High - Impacts basic input functionality

**Comment by YaLTeR** [comment]
Yes but this is a different issue. Please check for duplicates and if none, then open a new issue

**Comment by aohanhongzhi** [comment]
> Yes but this is a different issue. Please check for duplicates and if none, then open a new issue

https://github.com/YaLTeR/niri/issues/221 but it was closed

**Comment by YaLTeR** [comment]
Please post there

**Comment by YaLTeR** [comment]
Fixed in 4808ba2b2055a09008be17d3e9eeae2d592b7b18
